.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
```

Comment:
Probably want to add Linux shell scripts as well as they don't like the wrong line ending either